internal/pkg/util/env/clean.go

```diff
@@ -67,6 +68,17 @@ func SetContainerEnv(g *generate.Generator, env []string, cleanEnv bool, homeDes
 		}
 	}
 
+	if homeDest == "" {
+		// Image buid typically runs as root
```

Comment:
typo here, may want to change the comment when this code is moved to a slightly different context